package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "build": "rimraf lib && babel src/ -d lib/",
     "check-coverage": "babel-istanbul check-coverage",
-    "cover": "NODE_PATH=./src babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- --require test/main.js --recursive test",
+    "cover": "rimraf coverage && NODE_PATH=./src babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- --require test/main.js --recursive test",
```

Comment:
NIT: Do we need `./` prefixes here and on the `unit` task? If not, remove because they break windows.